src/artifact_choosing.rs

```diff
@@ -8,6 +8,8 @@ static PLATFORM_KEYWORDS: &[&str] = &["macos-x86_64", "darwin-x86_64", "macos",
 static PLATFORM_KEYWORDS: &[&str] = &[
     "macos-arm64",
     "darwin-arm64",
+    "macos-aarch64",
+    "darwin-aarch64",
     "macos-x86_64",
     "darwin-x86_64",
```

Comment:
Should we keep the `macos-x86_64` and `darwin-x86_64` here? I think we could just remove them

Reply:
Yeah that should be possible actually - since the fallback "macos" will be picked anyways

Reply:
I think it's fair to assume that if there are _any_ disambiguated binaries, they'll probably _all_ be disambiguated (rather than having `macos-arm64` and `macos`, you'd have `macos-arm64` and `macos-x86_64` to make sure that both names are clear).

So I'm onboard with that as well; don't need to do it in this PR if you don't want to, though.